content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -379,6 +379,7 @@ pipeline {
         stage('Example') {
             environment { /* <2> */
                 DEBUG_FLAGS = '-g'
+                FULL_CC_CMD = '${CC} ${DEBUG_FLAGS}' /* <3> */
```

Comment:
This example is confusing me, is `CC` supposed to be defined elsewhere or defined in the environment? I understand that `DEBUG_FLAGS` is coming from the other environment-set variable, but not where `CC` comes from.